packages/typespec-ts/test/modularUnit/scenarios/models/serialization/enumKeyNorm.md

```diff
@@ -0,0 +1,99 @@
+# Should generate enum key normalization cases
+
+## TypeSpec
+
+This is tsp definition.
+
+```tsp
+union ExtensibleString {
+  pascal: "pascal",
+  pascalCase: "pascalCase",
+  PascalCase: PascalCase,
+  pascalcase: "pascalcase",
+  Pascalcase: "Pascalcase",
+  pascal_case_: "pascal_case_",
+  pascal_case: "pascal_case",
+  _pascal_case: "_pascal_case",
+  `pascal, case`: "pascal, case",
+  MAX_of_MLD: "MAX_of_MLD",
+  ___pascal____case6666: "___pascal____case6666",
+  _10Pascal: "_10Pascal",
+  `090`: "090",
+  `10`: "10",
+  `1.0`: "1.0",
+  `-1.0`: "-1.0",
+  string,
```

Comment:
FYI - the normalization in HLC would be like:
![image](https://github.com/user-attachments/assets/ba4b6807-5dbf-46c7-901d-64794b21b8d6)